### ISSUE TITLE
Outdated WP.org screenshot of settings screen

### ISSUE THREAD

**Issue by dechov** [woocommerce/woocommerce#17178]
The first screenshot on the WP.org plugin page doesn't reflect the general settings in WooCommerce 3.2 — particularly the missing Store Address added in https://github.com/woocommerce/woocommerce/pull/15636 but also the REST API option and tab order.

![first screenshot](https://github.com/woocommerce/woocommerce/blob/5985089d4988098f016c580b27c60fc7b32c08f9/.wordpress-org/screenshot-1.png?raw=true)

**Comment by kloon** [comment]
Fairly easy fix, simply replace https://github.com/woocommerce/woocommerce/blob/master/.wordpress-org/screenshot-1.png with a new version.
